src/xSTIR/cSTIR/cstir.cpp

```diff
@@ -590,7 +590,7 @@ void* cSTIR_getAcquisitionDataDimensions(const void* ptr_acq, size_t ptr_dim)
 		SPTR_FROM_HANDLE(PETAcquisitionData, sptr_ad, ptr_acq);
 		dim[0] = sptr_ad->get_num_tangential_poss();
 		dim[1] = sptr_ad->get_num_views();
-		dim[2] = sptr_ad->get_num_non_tof_sinograms();
+		dim[2] = sptr_ad->get_num_non_TOF_sinograms();
```

Comment:
I'm confused as the STIR TOF PR https://github.com/UCL/STIR/pull/304 uses all lower case.
https://github.com/UCL/STIR/blob/0f0cccb30c2abe56f6fb2cb2ff6dbf4ade4cb1d4/src/include/stir/ProjData.h#L332

Reply:
ok. this is because of what I said [above](https://github.com/SyneRBI/SIRF/pull/543#pullrequestreview-355569887). However, we don't seem to have this function currently in SIRF, so I can't see how this was supposed to work.

My feeling is that the internal code can rely on STIR naming, so possibly easiest way around this problem is to go back to `get_num_non_tof_sinograms` here, and not add the function `get_num_non_TOF_sinograms` in SIRF

Reply:
ok, well, that wouldn't work either. So I guess we have to do what was said above. I'll try to do this now as it in a bit of a mess at the moment